packages/notifications/push/amplify_push_notifications/android/src/main/kotlin/com/amazonaws/amplify/amplify_push_notifications/AmplifyPushNotificationsPlugin.kt

```diff
@@ -42,6 +42,12 @@ open class AmplifyPushNotificationsPlugin : FlutterPlugin, ActivityAware,
          * Flutter API interface.
          */
         var flutterApi: PushNotificationsFlutterApi? = null
+
+        /**
```

Comment:
```suggestion
        /** Map of notification data when app was launched by tapping on the notification when the app is in a killed state, null otherwise. 
```